### ISSUE TITLE
loadAttributes fail

### ISSUE THREAD

**Issue by smolleyes** [tonistiigi/mega#23]
hi

don t know if mega made an update but some links can t be decoded

File.prototype._setAttributes = function(at, cb)  seems not working (sorry for file linked but can t find samll exemple file with this error)

exemple #!Vs4XxRRD!cK_gNBp-7SYmr-riEqcgRiUqllsIcShTo2yDyEahdwE

give:
undefined
SyntaxError {} undefined

thanks

**Comment by tonistiigi** [comment]
Yes, Links encrypted with folder key do not work at the moment. Only files with user key do.

I'll fix it the next time I have a chance to work on this lib.

#19

**Comment by tonistiigi** [comment]
Didn't notice the #F!. This may be something else. Leave this issue open for now.

**Comment by smolleyes** [comment]
hi

oh ok usually folders links are #F! strange :p

thanks

**Comment by smolleyes** [comment]
hi tony

more and more files have this bug :(

i confirm the problem is in (i think)

File.getCipher(this.key).decryptCBC(at) it return undefined (normal?)

or in the File.unpackAttributes = function(at) { in file.js

i found this small code working on same links

http://hacktracking.blogspot.fr/2013/07/download-mega-files-from-command-line.html

thanks :)

++

**Comment by smolleyes** [comment]
hi

many issues are appearing now with mega :( can t decode folder liks files any more, many links won t load attributes .... 

if you can have a look in the future toni.. thanks :)

**Comment by cameron5906** [comment]
Did you find any solution to this? Really worried that this lib isn't going to work for my project and it's unfortunately the only one out there with this kind of capability!
